ISLE/isleapp.cpp

```diff
@@ -540,6 +545,51 @@ void IsleApp::LoadConfig()
 
 	dictionary* dict = iniparser_load(iniConfig);
 
+	// [library:config]
```

Comment:
The logic does not seem correct to me.
When you run `isle --ini path/to/nonexisting/config.ini`, I'd expect the app to log an error that the ini you passed does not exist instead of creating a new one from scratch.

Reply:
That makes sense. I forgot we have the case where we explicitly specify a file on the command line. I agree in that case it should error out when it's missing. When no file is specified or found in a default path, only then should a file be created.